examples/chart/teleport-cluster/values.yaml

```diff
@@ -30,6 +30,23 @@ kubeClusterName: ""
 # If you want to run Teleport version X, you should use `helm --version X` instead.
 teleportVersionOverride: ""
 
+# The `proxyProtocol` value controls whether the Proxy and Auth pods will
+# use the PROXY protocol to receive the client's IP address when they are
+# behind a L4 load balancer (e.g. AWS ELB, GCP L4 LB, etc) with PROXY protocol
+# enabled. Since L4 LBs do not preserve the client's IP address, PROXY protocol is
+# required to ensure that Teleport can properly audit the client's IP address.
+#
+# When Teleport pods are not behind a L4 LB with PROXY protocol enabled, this
+# value should be set to "off" to avoid security issues where a malicious attacker
+# could spoof their IP address by prefixing their connection with a PROXY header.
+# Possible values are "on" and "off".
+#
+# - "on" will enable the PROXY protocol for all connections and will require the
+#   L4 LB to send a PROXY header.
+# - "off" will disable the PROXY protocol for all connections and denies all
+#   connections prefixed a PROXY header.
+# proxyProtocol: on
```

Comment:
IIRC Teleport defaults to `proxy_protocol: on` - should we uncomment this value by default to make the behaviour clearer?

Reply:
It no longer defaults to on.
https://github.com/gravitational/teleport-private/pull/991

Reply:
Oh yikes, OK